ctools/defragment_sharded_collection.py

```diff
@@ -666,9 +666,16 @@ def get_chunk_size_or_0(ch):
 
             # avoid moving larger chunks
             center_size_kb = await get_chunk_size(c)
+
             # Use < 0.6 so that we do not move chunks which were split before
-            if center_size_kb > target_chunk_size_kb * args.small_chunk_frac:
+            has_local_est_size = True if 'defrag_collection_est_size' in c else False
+            if has_local_est_size and center_size_kb > target_chunk_size_kb * args.small_chunk_frac:
```

Comment:
In order to highlight the important condition here and to not repeat it I would do:
```
if center_size_kb > target_chunk_size_kb * args.small_chunk_frac:
   if has_local_est_size:
      break
   else:
      continue
```